components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -331,7 +332,13 @@ const HostVirtualCards = props => {
         <P>
           <FormattedMessage
             id="Host.VirtualCards.List.Description"
-            defaultMessage="You can now manage and distribute Virtual Cards created on Privacy.com directly on Open Collective. You can assign multiple virtual cards to one collective. Virtual Cards enable quicker transactions, making disbursing money a lot easier!"
+            defaultMessage="You can now manage and distribute Virtual Cards created on Privacy.com directly on Open Collective. You can assign multiple virtual cards to one collective. Virtual Cards enable quicker transactions, making disbursing money a lot easier! {learnMoreLink}"
```

Comment:
Actually, the getI18nLink returns a link tag so you can use in the message:
```suggestion
            defaultMessage="You can now manage and distribute Virtual Cards created on Privacy.com directly on Open Collective. You can assign multiple virtual cards to one collective. Virtual Cards enable quicker transactions, making disbursing money a lot easier! <learnMoreLink>Learn more</learnMoreLink>"
```
This way the user is able to translate the anchor message "Learn more" to whatever they need.

Reply:
Ops, thanks for pointing it out. 😄